src/collider/Geom.cpp

```diff
@@ -90,17 +90,17 @@ std::vector<CollisionContact> Geom::Collide(Geom *b) const
 	return contacts;
 }
 
-static AABBd rotateAaabb(const AABBd &a, const matrix4x4d transA)
+static AABBd rotateAaabb(const AABBd &a, const matrix4x4d &transA)
```

Comment:
Little bit of an extra 'a' in 'Aaabb' here - probably my bug.

Reply:
Also if you're interested in implementing it, [this trick](https://zeux.io/2010/10/17/aabb-from-obb-with-component-wise-abs/) should give a bit of a speedup in constructing the AABB bounds for the oriented-bounding-box `transA * a`.

Reply:
I fixed the name typo, and implemented that neat little trick